hooks/post_gen_project.py

```diff
@@ -15,13 +16,34 @@ def main():
     project_name = "{{ cookiecutter.project_name }}"
     project_slug = "{{ cookiecutter.project_slug }}"
 
-    resource_name = "{{cookiecutter.resource_name}}"
-    resources = ["user", "customer", "order"]
+    read_from_file = {{ cookiecutter._read_from_file }}
+
+    resources_name = "{{ cookiecutter.resource_name }}"
+
+    yaml_file = "../hid-project-generator/config.yaml"
+
+    try:
+        if read_from_file is True:
+            with open(yaml_file, 'r') as stream:
+                output = yaml.safe_load(stream)
+                resources_name = output['resource_name']
+        else:
+            resources_name = resources_name.split(" ")
+    except FileNotFoundError as ex:
+        print("File not found")
```

Comment:
@akkumar29  Can we add logging instead of print statement ?

Reply:
Done